app/views/coronavirus_local_restrictions/_christmas_rules.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render "govuk_publishing_components/components/heading", {
+    text: t("coronavirus_local_restrictions.results.christmas_rules.heading"),
+    font_size: "m",
+    margin_bottom: 4,
+    mobile_top_margin: true,
+  } %>
+
+<p class="govuk-body">
+  <%= t("coronavirus_local_restrictions.results.christmas_rules.body") %>
+</p>
+
+<a href="<%= t("coronavirus_local_restrictions.results.christmas_rules.guidance.link") %>"
+  class="govuk-button govuk-button--secondary govuk-!-margin-bottom-8" role="button" data-module="govuk-button">
+  <%= t("coronavirus_local_restrictions.results.christmas_rules.guidance.label") %>
+</a>
```

Comment:
Can we use the button component for this?

Reply:
That is the plan but at the moment the button component doesn't support this style of a secondary button.

Once the support has been added in, this will be replaced by the component

Reply:
> Code looks good - it looks like you have some errors in the commit message to resolve. Also while you're editing the commit message, you might want to think more about explaining the why of some of the decisions you've made in this change - there seems to be some supplementary information about dates in there whereas this code doesn't seem to do anything with dates?
> 
> Just a minor thing but a screenshot would have been helpful in the description, as otherwise I had to run the code manually to see what it looked like (as the heroku version doesn't work for local restrictions).

Hey @kevindew - I've updated the things that you mentioned, happy to chat if you need to talk further.

Reply:
The HTML has been removed as support for this styling has now been added to the component